classes/package.py

```diff
@@ -18,6 +18,8 @@ def __init__(self, name):
         self.__version = ''
         self.__license = ''
         self.__src_url = ''
+        self.foo = "it should error out on this super long      asa                                                                        line"
+        import foo
```

Comment:
foo says:

[W0612(unused-variable), Package.__init__] Unused variable 'foo'
[E0401(import-error), Package.__init__] Unable to import 'foo'
```